src/edgepi/digital_output/digital_output_constants.py

```diff
@@ -15,3 +15,11 @@ class DoutPins(Enum):
     DOUT6 = 'DOUT6'
     DOUT7 = 'DOUT7'
     DOUT8 = 'DOUT8'
+
+@unique
+class DoutTriState(Enum):
+    """Digital Output Tri-state Enum"""
+
+    Z = -1
```

Comment:
without the comment in the README, i wouldn't know what Z is. now, i'm not sure what -1 means? is this just a state indicating high impedance?

Reply:
c64b58fe15dd94139db7e678f51a964dd20c790c

https://en.wikipedia.org/wiki/Three-state_logic

changed Z to HI_Z indicating the high impedance state of the gpio